KarhooUISDK/Screens/QuoteList/QuoteListPresenter.swift

```diff
@@ -97,9 +99,23 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
     }
 
     func getNumberOfResultsForQuoteFilters(_ filters: [QuoteListFilter]) -> Int {
+        handleLuggageAndPassengerFilters(filters)
```

Comment:
To avoid additional calls without any clear reason, this call should be fired once a filter is selected and accepted. In this case, it would be `selectedQuoteFilters` method.